platforms/documentation/docs/src/docs/userguide/authoring-builds/plugins/implementing_gradle_plugins_binary.adoc

```diff
@@ -699,10 +699,10 @@ When it comes to reporting problems, we support two different modes:
 
 For more details, see the link:{javaDocPath}/org/gradle/api/problems/ProblemReporter.html[ProblemReporter] documentation.
 
-=== Problem aggregation
+=== Problem summarization
 
-When reporting problems, Gradle will aggregate similar problems by sending them through the Tooling API based on the problem's category label.
+When reporting problems, Gradle will summarize similar problems before sending them to any progress event listener, including the  link:{javaDocPath}/org/gradle/tooling/events/ProgressListener.html[Tooling API ProgressListener].
```

Comment:
I think we should phras this in a more user-centric way:

When reporting problems, Gradle makes sure that the report is not too redundant. In other words, it will not report problems with the same if some threshold is reached.

Reply:
(please don't copy-paste my text verbatim, it can probably use some authoring)